extensions/radix/src/primitives.tsx

```diff
@@ -59,19 +59,19 @@ const primitivesRoutes = [
       { title: "Dialog", slug: "docs/primitives/components/dialog" },
       { title: "Dropdown Menu", slug: "docs/primitives/components/dropdown-menu" },
       { title: "Hover Card", slug: "docs/primitives/components/hover-card" },
-      { title: "Hover Menu", slug: "docs/primitives/components/hover-menu", draft: true },
       { title: "Label", slug: "docs/primitives/components/label" },
       { title: "Menubar", slug: "docs/primitives/components/menubar", draft: true },
+      { title: "Navigation Menu", slug: "docs/primitives/components/navigation-menu", beta: true },
       { title: "Popover", slug: "docs/primitives/components/popover" },
       { title: "Progress", slug: "docs/primitives/components/progress" },
       { title: "Radio Group", slug: "docs/primitives/components/radio-group" },
       { title: "Scroll Area", slug: "docs/primitives/components/scroll-area" },
-      { title: "Select", slug: "docs/primitives/components/select", draft: true },
+      { title: "Select", slug: "docs/primitives/components/select", beta: true },
       { title: "Separator", slug: "docs/primitives/components/separator" },
       { title: "Slider", slug: "docs/primitives/components/slider" },
       { title: "Switch", slug: "docs/primitives/components/switch" },
       { title: "Tabs", slug: "docs/primitives/components/tabs" },
-      { title: "Toast", slug: "docs/primitives/components/toast", draft: true },
+      { title: "Toast", slug: "docs/primitives/components/toast", beta: true },
```

Comment:
if you want to rename from `draft` to `beta`, you gotta ensure the list filter is updated too https://github.com/raycast/extensions/blob/main/extensions/radix/src/primitives.tsx#L11

Reply:
Thanks for your comment, Pedro. The original list uses both `draft` _and_ `beta`, so this is not a rename. I figured we probably want to continue hiding drafts, but allow beta primitives to be visible. If you prefer hiding beta primitives as well, I can make that change.

Reply:
> The original list uses both draft and beta,

What do you mean by original list?

The primitives types currently in `main` never supported `beta`:
<img width="209" alt="CleanShot 2022-03-29 at 17 14 22@2x" src="https://user-images.githubusercontent.com/372831/160645122-47904798-75ec-4b86-a4fc-7e65f95a4ddc.png">

Reply:
>What do you mean by original list?
I'm referring to https://github.com/radix-ui/website/blob/main/lib/primitivesRoutes.ts. You can see that `beta` is now supported there. But I'm seeing that I didn't update the `PageProps` type accordingly, so I just pushed a fix for that.

Reply:
what im saying is that you added a new `beta` flag, but you're doing nothing with it.

the `draft` flag was used to filter out items from the list ([as you can see here](https://github.com/raycast/extensions/blob/main/extensions/radix/src/primitives.tsx#L11))

the `deprecated` flag is used to show an extra label, ([as you can see here](https://github.com/raycast/extensions/blob/main/extensions/radix/src/primitives.tsx#L16)) like this: 
![CleanShot 2022-04-04 at 11 05 17@2x](https://user-images.githubusercontent.com/372831/161511247-77aa485f-1025-4adf-8a75-927aa65e5904.png)

but the `beta` flag is not being used for anything.

I understand you took it from the radix-site repo, but in there it gets used:
<img width="290" alt="CleanShot 2022-04-04 at 11 06 26@2x" src="https://user-images.githubusercontent.com/372831/161511449-5cd887d5-8b4a-4281-b1fc-e9067b428859.png">


so, if you're gonna add a new flag, I think we need to use it in the UI as a subtitle prop.

Reply:
Oh I see, I completely missed that, thanks for clarifying! I just updated the pull request to include the "Beta" subtitle. Let me know if there's anything else that needs to be changed.